importer/views.py

```diff
@@ -0,0 +1,43 @@
+"""
+Views for the importer app.
+"""
+
+from django.shortcuts import render
+from django.contrib.auth.models import User
+
+from learningresources.models import Repository, Course, LearningResource
+from .forms import UploadForm
+
+
+def status(request):
+    """
+    Show what's currently in the system.
+    """
+    counts = {
+        "repos": Repository.objects.count(),
+        "courses": Course.objects.count(),
+        "loxes": LearningResource.objects.count(),
+    }
+    return render(
+        request,
+        "status.html",
+        counts,
+    )
+
+
+def upload(request):
+    """
+    Upload a OLX archive.
+    """
+    form = UploadForm()
+    if request.user.is_anonymous():
+        request.user, _ = User.objects.get_or_create(username="dirty_hack")
+    if request.method == "POST":
+        form = UploadForm(data=request.POST, files=request.FILES)
```

Comment:
Looks like we are missing a test case for this based on coverage, would be nice to have.